test/spec/ExtensionManager-test.js

```diff
@@ -829,6 +838,12 @@ define(function (require, exports, module) {
                 it("should start with the full set sorted in reverse publish date order", function () {
                     expect(model.filterSet).toEqual(["theme-1", "theme-2"]);
                 });
+                
+                it("should start with the full set sorted in reverse download count order", function () {
+                    PreferencesManager.set("extensions.sort", "downloadCount");
+                    model._setSortedExtensionList(ExtensionManager.extensions, true);
+                    expect(model.filterSet).toEqual(["theme-2", "theme-1"]);
+                });
```

Comment:
A simple check could be added that Brackets actually renders the `totalDownloads` to the UI 👍